hector_mapping/src/HectorMappingRos.cpp

```diff
@@ -157,17 +157,18 @@ HectorMappingRos::HectorMappingRos()
       tmp.dynamicMapServiceServer_ = node_.advertiseService("dynamic_map", &HectorMappingRos::mapCallback, this);
     }
 
-    // Initialize services to reset map, and toggle scan pause
-    reset_map_service_ = node_.advertiseService("reset_map", &HectorMappingRos::resetMapCallback, this);
-    toggle_scan_processing_service_ = node_.advertiseService("pause_mapping", &HectorMappingRos::pauseMapCallback, this);
```

Comment:
my bad in the previous PR: I had placed this inside a `for` loop, so I'm bringing it out of that loop